.github/workflows/canary-aarch64-linux.yml

```diff
@@ -66,6 +66,8 @@ jobs:
           rm -rf /root/.djl.ai/
           DJL_ENGINE=pytorch-native-auto PYTORCH_VERSION=2.4.0 ./gradlew clean run
```

Comment:
We don't really need to support 2.4.0 in 0.31.0

Reply:
2.4.0 is published in nightly_publish, should we remove it from nightly_publish first?

Reply:
I think we can remove 1.13.1, since it's pretty old.

Reply:
We need keep 1.13.1, that for cuda11 and last version support AL2
We also need keep 2.3.1 that for cu121

Reply:
ok

Reply:
We will remove it in the next release.